schema/assets/ui.html

```diff
@@ -7,12 +7,19 @@
   <body>
     <rapi-doc
       spec-url="/schema/swagger.json"
-      render-style="view"
+      render-style="focused"
       schema-style="table"
+      default-schema-tab="schema"
+      schema-description-expanded="true"
       allow-authentication="true"
       allow-server-selection="false"
+      allow-spec-file-download="true"
       show-header="false"
-      schema-description-expanded="true"
-      theme="dark"></rapi-doc>
+      primary-color="#ffb901"
+      nav-bg-color="#ffc11e"
```

Comment:
```suggestion
      nav-bg-color="#ffc11e"
      nav-text-color="#000000"
```
The navbar text color looks a bit washed out.

Before
---
<img width="170" alt="image" src="https://github.com/cerbos/cerbos/assets/3899719/40190fb7-6160-4f0a-8f43-51abe445675d">

After
---
<img width="169" alt="image" src="https://github.com/cerbos/cerbos/assets/3899719/1cc4570a-33ca-45b1-9179-ac5bfacc1661">